### ISSUE TITLE
InputFieldChanged event is not emited when input field becomes invalid

### ISSUE THREAD

**Issue by takarogabor** [kekeh/ngx-mydatepicker#4]
I'd like to use this great datepicker however I found a bug. 

I can't set my form to invalid state when text in the input field becomes invalid because InputFieldChanged event is not emitted.

How to reproduce:
1. Type (not pick) a valid date into the input element eg. '2017-05-07'
2. Delete the last char

It should emit an InputFieldChanged event with valid: false but it doesn't emit any event.

**Comment by takarogabor** [comment]
I've did some debugging. I think the problem is in the [ onKeyUp](https://github.com/kekeh/ngx-mydatepicker/blob/master/src/ngx-my-date-picker/ngx-my-date-picker.input.ts#L69), it should save the inputText when the input is valid also.
```javascript
if (date.day !== 0 && date.month !== 0 && date.year !== 0) {
  this.inputText = this.elem.nativeElement.value;
```

**Comment by kekeh** [comment]
There was a bug. I fixed it in version 0.0.7. Thanks!

**Comment by takarogabor** [comment]
Thank you for the quick fix!
